src/features/workspaces/workspace-detail/components/GroupDetailsDrawer.tsx

```diff
@@ -207,7 +207,12 @@ export const GroupDetailsDrawer: React.FC<GroupDetailsDrawerProps> = ({
 
     // Map roles to rows with conditional inheritance information
     const roleRows = roles.map((role) => {
-      const baseRow: (string | React.ReactElement)[] = [role.display_name || role.name || ''];
+      const roleLink = (
+        <a href={`/iam/user-access/roles/detail/${role.uuid}`} className="pf-v5-c-button pf-m-link pf-m-inline">
```

Comment:
should we use the PF link instead of anchor tag?

Reply:
there is an `AppLink` component that deals with prefixing the link with the right bit, so it should look like


```suggestion
        <AppLink to={`/roles/detail/${role.uuid}`} className="pf-v5-c-button pf-m-link pf-m-inline">
```